<html>
<head>

<style>
	table {border-collapse: collapse; padding-bottom:5px; margin-bottom:5px;}
	table, th, td {border: 1px solid #CCCCCC;}
	td, th {padding:2px;}
	.man {background-color:#FFFFCC;}
	ul {padding: 0px; padding-bottom:3px; padding-top:0px; margin: 0px;}
	ol {padding: 0px; padding-bottom:3px; padding-top:0px; margin: 0px;}
	li {padding-left:2px; padding-top:0px; padding-bottom:0px; margin-left:25px;}
	p {margin:2px 0px;}
	.highlightcolor { background-color:#D8BFD8; }
</style>



<!-- rmb toggle -->


<script>
function toggle1() {
 if( document.getElementById("hidethis1").style.display=='none' ){
   document.getElementById("hidethis1").style.display = '';
 }else{
   document.getElementById("hidethis1").style.display = 'none';
 }
}

function toggle2() {
 if( document.getElementById("hidethis2").style.display=='none' ){
   document.getElementById("hidethis2").style.display = '';
 }else{
   document.getElementById("hidethis2").style.display = 'none';
 }
}

function toggle3() {
 if( document.getElementById("hidethis3").style.display=='none' ){
   document.getElementById("hidethis3").style.display = '';
 }else{
   document.getElementById("hidethis3").style.display = 'none';
 }
}
function toggle4() {
 if( document.getElementById("hidethis4").style.display=='none' ){
   document.getElementById("hidethis4").style.display = '';
 }else{
   document.getElementById("hidethis4").style.display = 'none';
 }
}
</script>

    <title>beta3 YES and NO Words for FP and FN experiments</title>


<STYLE TYPE="text/css">
<!--
TH{font-family: Verdana; font-size: 7pt; font-weight: bold;}
TD{font-family: Verdana; font-size: 7pt;}
--->
</STYLE>

</head>


</head>

<body>

<h3>TFIDF VECTORIZER w/ beta=3<br>
YES, NO words in False Positives and False Negatives</h3><p>
<p>

<font size=-1>


<b>TFIDF VECTORIZER w/ beta=3</b><p>

<OL>
<LI><b>YES and NO Features from the Lexicon Histogram (ordered by rank)</b><p>

<font color='#0000A0'><span onClick="toggle1();" onMouseOver="this.style.color = '#cc0000'; this.style.backgroundColor='#FFFFFF';" onMouseOut="this.style.color ='#0000A0'; this.style.backgroundColor='#FFFFFF';"><font size="3"><b>Show/Hide - YES and NO Features</b></span></font></font>
<p>

<div id="hidethis1" style="display:none;">

<blockquote>
<TABLE border=1>
<TR><TH bgcolor="#ffe6e6">NO Features</th></tr>
<tr><td bgcolor="#ffe6e6">cells</td></tr><tr><td bgcolor="#ffe6e6">series</td></tr><tr><td bgcolor="#ffe6e6">superseries</td></tr><tr><td bgcolor="#ffe6e6">composed</td></tr><tr><td bgcolor="#ffe6e6">superseries composed</td></tr><tr><td bgcolor="#ffe6e6">individual</td></tr><tr><td bgcolor="#ffe6e6">individual series</td></tr><tr><td bgcolor="#ffe6e6">refer </td></tr><tr><td bgcolor="#ffe6e6">refer individual</td></tr><tr><td bgcolor="#ffe6e6">refer individual series</td></tr><tr><td bgcolor="#ffe6e6">following</td></tr><tr><td bgcolor="#ffe6e6">cell</td></tr><tr><td bgcolor="#ffe6e6">treated</td></tr><tr><td bgcolor="#ffe6e6">induced</td></tr><tr><td bgcolor="#ffe6e6">subset series</td></tr><tr><td bgcolor="#ffe6e6">following subset</td></tr><tr><td bgcolor="#ffe6e6">superseries composed following</td></tr><tr><td bgcolor="#ffe6e6">composed following</td></tr><tr><td bgcolor="#ffe6e6">following subset series</td></tr><tr><td bgcolor="#ffe6e6">composed following subset</td></tr><tr><td bgcolor="#ffe6e6">subset</td></tr><tr><td bgcolor="#ffe6e6">facs</td></tr><tr><td bgcolor="#ffe6e6">subseries</td></tr><tr><td bgcolor="#ffe6e6">composed subseries listed</td></tr><tr><td bgcolor="#ffe6e6">subseries listed</td></tr><tr><td bgcolor="#ffe6e6">composed subseries</td></tr><tr><td bgcolor="#ffe6e6">superseries composed subseries</td></tr><tr><td bgcolor="#ffe6e6">listed</td></tr><tr><td bgcolor="#ffe6e6">listed refer individual</td></tr><tr><td bgcolor="#ffe6e6">subseries listed refer</td></tr>
<TR><th bgcolor="#e6e6ff">YES Features</th></tr>
<tr><td bgcolor="#e6e6ff">testis</td></tr><tr><td bgcolor="#e6e6ff">dissected</td></tr><tr><td bgcolor="#e6e6ff">genotype</td></tr><tr><td bgcolor="#e6e6ff">pooled</td></tr><tr><td bgcolor="#e6e6ff">postnatal</td></tr><tr><td bgcolor="#e6e6ff">embryos</td></tr><tr><td bgcolor="#e6e6ff">embryonic day</td></tr><tr><td bgcolor="#e6e6ff">deleted</td></tr><tr><td bgcolor="#e6e6ff">organs</td></tr><tr><td bgcolor="#e6e6ff">produced</td></tr><tr><td bgcolor="#e6e6ff">wild</td></tr><tr><td bgcolor="#e6e6ff">species</td></tr><tr><td bgcolor="#e6e6ff">individuals</td></tr><tr><td bgcolor="#e6e6ff">litermate</td></tr><tr><td bgcolor="#e6e6ff">development</td></tr><tr><td bgcolor="#e6e6ff">hearts</td></tr><tr><td bgcolor="#e6e6ff">gastrocnemius</td></tr><tr><td bgcolor="#e6e6ff">cryosections</td></tr><tr><td bgcolor="#e6e6ff">male</td></tr><tr><td bgcolor="#e6e6ff">expression developing</td></tr><tr><td bgcolor="#e6e6ff">design</td></tr><tr><td bgcolor="#e6e6ff">wt runx3 freshly</td></tr><tr><td bgcolor="#e6e6ff">runx3 freshly</td></tr><tr><td bgcolor="#e6e6ff">identify runx3 freshly</td></tr><tr><td bgcolor="#e6e6ff">runx3 responsive</td></tr><tr><td bgcolor="#e6e6ff">runx3 freshly isolated</td></tr><tr><td bgcolor="#e6e6ff">runx3 responsive genes</td></tr><tr><td bgcolor="#e6e6ff">mutant</td></tr><tr><td bgcolor="#e6e6ff">knockout</td></tr><tr><td bgcolor="#e6e6ff">mice microarray</td></tr>
</table></blockquote>
</div>
<br><br>


<LI><b>SGD Classifier with Tfidf Vectorizer selections</b><p>
cvect__max_df:0.98<br>
cvect__min_df:2<br>
cvect__ngram_range:(1, 3)<br>
sgd__alpha:0.1<br>
sgd__eta0:0.001<br>
sgd__learning_rate:'constant'<br>
sgd__loss:'log'<br>
sgd__n_iter:5<br>
sgd__penalty:'elasticnet'<br>
sgd__shuffle:True<br><br>



<LI><b>Test and Training Set Classification Report and Confusion Matrix</b><p>

<u>Test Set</u> Classification Report:<p>
<table>
<tr><th>class</th><th>precision</th><th>recall</th><th>f1-score</th><th>support</th></tr><tr><td>yes</td><td>0.62</td><td>0.97</td><td>0.76</td><td>134</td></tr><tr><td>no</td><td>0.99</td><td>0.83</td><td>0.90</td><td>461</td></tr><tr><td>avg/total</td><td>0.91</td><td>0.86</td><td>0.87</td><td>595</td></tr><tr><td colspan=5>F3: 0.918</td></tr>
</table>
<p>



<u>Test Set</u> Confusion Matrix:<p>
<table>
<tr><td>tp:<b>130</b></td><td>fn:<b>4</b></td></tr>
<tr><td>fp:<b>80</b></td><td>tn:<b>381</b></td></tr>
</table>
<br><br>


<u>Training Set</u> Classification Repot:<p>

<table>
<tr><th>class</th><th>precision</th><th>recall</th><th>f1-score</th><th>support</th></tr><tr><td>yes</td><td>0.73</td><td>1.00</td><td>0.84</td><td>352</td></tr><tr><td>no</td><td>1.00</td><td>0.91</td><td>0.95</td><td>1432</td></tr><tr><td>avg/total</td><td>0.95</td><td>0.93</td><td>0.93</td><td>1784</td></tr><tr><td colspan=5>F3: 0.964</td></tr>
</table>
<p>



<u>Training Set</u> Confusion Matrix:<p>
<table>
<tr><td>tp:<b>352</b></td><td>fn:<b>0</b></td></tr>
<tr><td>fp:<b>131</b></td><td>tn:<b>1301</b></td></tr>
</table>
<br><br>



<LI><b>Sample Queries for NO Word and YES Word Counts</b><p>
Queries for a False Negative (E-ERAD-283) shown.<p>


<blockquote>
<b>Sample Query for NO Word Count</b><br>
<font size=-1>
SQL <font size=-1>(postgres syntax)</font><br>
<textarea name="SQL" rows="10" cols="80">
select distinct hte._experiment_key, a.accid, hte.name, hte.description
into temp temp1
from gxd_htexperiment hte, acc_accession a
where a._mgitype_key = 42
and a._logicaldb_key = 189
and a.accid in ('E-ERAD-283', 'E-GEOD-36026', 'E-MTAB-3707', 'E-MTAB-1035')
and a._object_key = hte._experiment_key
||
CREATE INDEX idx1 ON temp1 (accid, name, description)
||
select accid, name, description
into temp temp2
from temp1
where accid = 'E-ERAD-283'
||
CREATE INDEX idx2 ON temp2 (accid, name)
||
SELECT accid, regexp_replace(name, '[^a-zA-Z0-9]+', ' ','g') as name
into temp temp3
from temp2
||
SELECT accid, name, 
       regexp_split_to_table(name, '\s') as NO_Word
into temp temp4
FROM temp3
||
select distinct accid, name
from temp2
||
select distinct NO_Word, count(*) as count_in_name
from temp4
where NO_Word ilike ANY(ARRAY['cells', 'series', 'superseries', 'composed', 'superseries composed', 'individual', 'individual series', 'refer ', 'refer individual', 'refer individual series', 'following', 'cell', 'treated', 'induced', 'subset series', 'following subset', 'superseries composed following', 'composed following', 'following subset series', 'composed following subset', 'subset', 'facs', 'subseries', 'composed subseries listed', 'subseries listed', 'composed subseries', 'superseries composed subseries', 'listed', 'listed refer individual', 'subseries listed refer'])
GROUP BY NO_Word
||
SELECT accid, regexp_replace(description, '[^a-zA-Z0-9]+', ' ','g') as description
into temp temp5
from temp2
||
SELECT accid, description, 
       regexp_split_to_table(description, '\s') as NO_Word
into temp temp6
FROM temp5
||
select distinct accid, description
from temp2
||
select distinct NO_Word, count(*) as count_in_desc
from temp6
where NO_Word ilike ANY(ARRAY['cells', 'series', 'superseries', 'composed', 'superseries composed', 'individual', 'individual series', 'refer ', 'refer individual', 'refer individual series', 'following', 'cell', 'treated', 'induced', 'subset series', 'following subset', 'superseries composed following', 'composed following', 'following subset series', 'composed following subset', 'subset', 'facs', 'subseries', 'composed subseries listed', 'subseries listed', 'composed subseries', 'superseries composed subseries', 'listed', 'listed refer individual', 'subseries listed refer'])
GROUP BY NO_Word
</textarea><p>





<b>Sample Query for YES Word Count</b><br>

<font size=-1>
SQL <font size=-1>(postgres syntax)</font><br>
<textarea name="SQL" rows="10" cols="80">
select distinct hte._experiment_key, a.accid, hte.name, hte.description
into temp temp1
from gxd_htexperiment hte, acc_accession a
where a._mgitype_key = 42
and a._logicaldb_key = 189
and a.accid in ('E-ERAD-283', 'E-GEOD-36026', 'E-MTAB-3707', 'E-MTAB-1035')
and a._object_key = hte._experiment_key
||
CREATE INDEX idx1 ON temp1 (accid, name, description)
||
select accid, name, description
into temp temp2
from temp1
where accid = 'E-ERAD-283'
||
CREATE INDEX idx2 ON temp2 (accid, name)
||
SELECT accid, regexp_replace(name, '[^a-zA-Z0-9]+', ' ','g') as name
into temp temp3
from temp2
||
SELECT accid, name, 
       regexp_split_to_table(name, '\s') as YES_Word
into temp temp4
FROM temp3
||
select distinct accid, name
from temp2
||
select distinct YES_Word, count(*) as count_in_name
from temp4
where YES_Word ilike ANY(ARRAY['testis', 'dissected', 'genotype', 'pooled', 'postnatal', 'embryos', 'embryonic day', 'deleted', 'organs', 'produced', 'wild', 'species', 'individuals', 'litermate', 'development', 'hearts', 'gastrocnemius', 'cryosections', 'male', 'expression developing', 'design', 'wt runx3 freshly', 'runx3 freshly', 'identify runx3 freshly', 'runx3 responsive', 'runx3 freshly isolated', 'runx3 responsive genes', 'mutant', 'knockout', 'mice microarray'])
GROUP BY YES_Word
||
SELECT accid, regexp_replace(description, '[^a-zA-Z0-9]+', ' ','g') as description
into temp temp5
from temp2
||
SELECT accid, description, 
       regexp_split_to_table(description, '\s') as YES_Word
into temp temp6
FROM temp5
||
select distinct accid, description
from temp2
||
select distinct YES_Word, count(*) as count_in_desc
from temp6
where YES_Word ilike ANY(ARRAY['testis', 'dissected', 'genotype', 'pooled', 'postnatal', 'embryos', 'embryonic day', 'deleted', 'organs', 'produced', 'wild', 'species', 'individuals', 'litermate', 'development', 'hearts', 'gastrocnemius', 'cryosections', 'male', 'expression developing', 'design', 'wt runx3 freshly', 'runx3 freshly', 'identify runx3 freshly', 'runx3 responsive', 'runx3 freshly isolated', 'runx3 responsive genes', 'mutant', 'knockout', 'mice microarray'])
GROUP BY YES_Word
</textarea><p>
</blockquote>
<br><br>



<LI><b>False Negative Word Counts (NO/YES Words)</b><p>

Number of False Negatives:  4<br>
E-ERAD-283<br>
E-GEOD-36026<br>
E-MTAB-3707<br>
E-MTAB-1035<br><br>

False Negative: <b>E-ERAD-283</b><p>


<TABLE border=1>
<TR><TH>accid<TH>name
<TR><TD>E-ERAD-283<TD>Single cell transcriptomics of CASTxB6 F1 mouse olfactory and liver cells
</TABLE>
<p>
<TABLE border=1>
<TR><TH>NO_Word<TH>count_in_name
<TR><TD>cell<TD>1
<TR><TD>cells<TD>1
<TR><TH>YES_Word<TH>count_in_name
<TR><TD colspan=2>0 YES Words in name
</table>
<p>

<TABLE border=1>
<TR><TH>accid<TH>description
<TR><TD>E-ERAD-283<TD>High-throughput sequencing of individual liver cells from adult B6Cast F1 female mice.This data is part of a pre-publication release. For information on the proper use of pre-publication data shared by the Wellcome Trust Sanger Institute (including details of any publication moratoria), please see http://www.sanger.ac.uk/datasharing/
</TABLE>


<TABLE border=1>
<TR><TH>NO_word<TH>count_in_desc
<TR><TD>cells<TD>1
<TR><TD>individual<TD>1
<TR><TH>YES_Word<TH>count_in_desc
<TR><TD colspan=2>0 YES Words in desc
</TABLE>
<br><br>


False Negative: <b>E-GEOD-36026</b><p>


<TABLE border=1>
<TR><TH>accid<TH>name
<TR><TD>E-GEOD-36026<TD>RNA-seq from ENCODE/LICR
</TABLE>
<p>
<TABLE border=1>
<TR><TH>NO_Word<TH>count_in_name
<TR><TD colspan=2>0 NO Words in name
<TR><TH>YES_Word<TH>count_in_name
<TR><TD colspan=2>0 YES Words in name
</table>
<p>

<TABLE border=1>
<TR><TH>accid<TH>description
<TR><TD>E-GEOD-36026<TD>Using RNA-Seq (Mortazavi et al., 2008), high-resolution genome-wide maps of the mouse transcriptome across multiple mouse (C57Bl/6) tissues and primary cells were generated. Cells were grown according to the approved ENCODE cell culture protocols (http://genome.ucsc.edu/ENCODE/protocols/cell/mouse). RNA-Seq RNA samples from tissues and primary cells were extracted from Trizol according to protocol (Invitrogen). PolyA+ RNA was purified with the Dynabeads mRNA purification kit (Invitrogen). The mRNA libraries were prepared for strand-specific sequencing as described previously (Parkhomchuk et al., 2009). Sequencing and Analysis Samples were sequenced on Illumina Genome Analyzer II, Genome Analyzer IIx and HiSeq 2000 platforms for 36 cycles. Image analysis, base calling and alignment to the mouse genome version mm9 were performed using Illumina's RTA. Alignment to the mouse genome was performed using TopHat (Trapnell et al., 2009). Wig files were generated by TopHat and expression levels were calculated with Cufflinks (Trapnell et al., 2010). 
</TABLE>


<TABLE border=1>
<TR><TH>NO_Word<TH>count_in_desc
<TR><TD>cells<TD>3
<TR><TD>cell<TD>2
<TR><TH>YES_Word<TH>count_in_desc
<TR><TD colspan=2>0 YES Words in desc
</TABLE>
<br><br>



False Negative: <b>E-MTAB-3707</b><p>


<TABLE border=1>
<TR><TH>accid<TH>name
<TR><TD>E-MTAB-3707<TD>Gene expression profiling of mouse liver cells
</TABLE>
<p>
<TABLE border=1>
<TR><TH>NO_Word<TH>count_in_name
<TR><TD>cells<TD>1
<TR><TH>YES_Word<TH>count_in_name
<TR><TD colspan=2>0 YES Words in name
</table>
<p>

<TABLE border=1>
<TR><TH>accid<TH>description
<TR><TD>E-MTAB-3707<TD>We generated the individual transcriptomes of 96 liver cells using the Fluidigm C1 platform. In brief, a suspension of cells was prepared from the liver of a 14-week old B6CastF1 (C57Bl/6J mother x CAST/Ei father) female mouse and loaded onto a 10-17 m C1 Single-Cell Auto Prep IFC (Fluidigm), and cell capture was performed according to the manufacturers instructions. 
</TABLE>


<TABLE border=1>
<TR><TH>NO_Word<TH>count_in_desc
<TR><TD>cells<TD>2
<TR><TD>cell<TD>2
<TR><TD>individual<TD>1
<TR><TH>YES_Word<TH>count_in_desc
<TR><TD colspan=2>0 YES Words in desc
</TABLE>
<br><br>




False Negative: <b>E-MTAB-1035</b><p>


<TABLE border=1>
<TR><TH>accid<TH>name
<TR><TD>E-MTAB-1035<TD>Transcription profiling by array of differentiating primary primitive and definitive mouse red blood cells 
</TABLE>
<p>
<TABLE border=1>
<TR><TH>NO_Word<TH>count_in_name
<TR><TD>cells<TD>1
<TR><TH>YES_Word<TH>count_in_name
<TR><TD colspan=2>0 YES Words in name
</table>
<p>

<TABLE border=1>
<TR><TH>accid<TH>description
<TR><TD>E-MTAB-1035<TD>Delineate gene anatomy of differentiating primary primitive and definitive RBCs
</TABLE>


<TABLE border=1>
<TR><TH>NO_Word<TH>count_in_desc
<TR><TD colspan=2>0 NO Words in desc
<TR><TH>YES_Word<TH>count_in_desc
<TR><TD colspan=2>0 YES Words in desc
</TABLE>
<br><br>







<LI><b>False Positive Word Counts (NO/YES Words)</b><p>

Number of False Positives:  80<br>
E-GEOD-38001<br>
E-GEOD-769<br>
E-GEOD-77846<br>
E-MTAB-3814<br>
E-GEOD-31013<br>
E-GEOD-2882<br>
E-GEOD-9033<br>
E-GEOD-47735<br>
E-GEOD-47395<br>
E-GEOD-15729<br>
E-GEOD-66420<br><br>

False Positive: <b>E-GEOD-38001</b><p>


<TABLE border=1>
<TR><TH>accid<TH>name
<TR><TD>E-GEOD-38001<TD>Transcription profiling by array of mouse wild type and Htt mutant striatal cells upon heat shock 
</TABLE>
<p>
<TABLE border=1>
<TR><TH>NO_Word<TH>count_in_name
<TR><TD>cells<TD>1
<TR><TH>YES_Word<TH>count_in_name
<TR><TD>mutant<TD>1
<TR><TD>wild<TD>1
</table>
<p>

<TABLE border=1>
<TR><TH>accid<TH>description
<TR><TD>E-GEOD-38001<TD>In Huntington's disease (HD), polyglutamine expansions in the huntingtin (Htt) protein cause subtle changes in cellular functions that, over-time, lead to neurodegeneration and death. Studies have indicated that activation of the heat shock response can reduce many of the effects of mutant Htt in disease models, suggesting that the heat shock response is impaired in the disease. To understand the basis for this impairment, we have used genome-wide chromatin immunoprecipitation followed by massively parallel sequencing (ChIP-Seq) to examine the effects of mutant Htt on the master regulator of the heat shock response, HSF1. We find that, under normal conditions, HSF1 function is highly similar in cells carrying either wild-type or mutant Htt. However, polyQ-expanded Htt severely blunts the HSF1-mediated stress response. Surprisingly, we find that the HSF1 targets most affected upon stress are not directly associated with proteostasis, but with cytoskeletal binding, focal adhesion and GTPase activity. Our data raise the intriguing hypothesis that the accumulated damage from life-long impairment in these stress responses may contribute significantly to the etiology of Huntington's disease. Affymetrix MG430 2.0 expression levels of wild-type (STHdhQ7/Q7) and mutant (STHdhQ111/Q111) striatal cells under growth condition (33 C) and upon heat shock (42 C for six hours) 
</TABLE>


<TABLE border=1>
<TR><TH>no_word<TH>count_in_desc
<TR><TD>cells<TD>2
<TR><TH>YES_Word<TH>count_in_desc
<TR><TD>mutant<TD>4
<TR><TD>wild<TD>2
</TABLE>
<br><br>


False Positive: <b>E-GEOD-769</b><p>




<TABLE border=1>
<TR><TH>accid<TH>name
<TR><TD>E-GEOD-769<TD>Transcription profiling of pancreases of cystic fibrosis mouse with a genotype of gastrin+/-, CFTR-/- to study the gene expression patterns in cystic fibrosis 
</TABLE>
<p>
<TABLE border=1>
<TR><TH>NO_Word<TH>count_in_name
<TR><TD colspan=2>0 NO Words in name
<TR><TH>YES_Word<TH>count_in_name
<TR><TD>genotype<TD>1
</table>
<p>



<TABLE border=1>
<TR><TH>accid<TH>description
<TR><TD>E-GEOD-769<TD>Total RNA was prepared using TRIzol reagent from the pancreata of eight week old male mice. The genotypes were Control: gastrin+/-, CFTR+/+; and CF: gastrin+/-, CFTR-/-. All mice were on 95% black6, 5% 129Sv background. Mice were fed Peptamen from age 10 days to prevent intestinal obstruction.
</table>


<TABLE border=1>
<TR><TH>NO_Word<TH>count_in_desc
<TR><TD colspan=2>0 NO Words in desc
<TR><TH>YES_Word<TH>count_in_desc
<TR><TD>male<TD>1
</TABLE>
<br><br>



False Positive: <b>E-GEOD-77846</b><p>


<TABLE border=1>
<TR><TH>accid<TH>name
<TR><TD>E-GEOD-77846<TD>Transcription profiling by high throughput sequencing of c-KIT+ splenocytes from murine acute myeloid leukemia and myeloproliferative neoplasm 
</TABLE>
<p>
<TABLE border=1>
<TR><TH>NO_Word<TH>count_in_name
<TR><TD colspan=2>0 NO Words in name
<TR><TH>YES_Word<TH>count_in_name
<TR><TD colspan=2>0 YES Words in name
</table>
<p>

<TABLE border=1>
<TR><TH>accid<TH>description
<TR><TD>E-GEOD-77846<TD>Cytogenetically normal acute myeloid leukemia (CN-AML) represents nearly 50% of human acute myeloid leukemia (AML) cases with a 5-year overall survival of approximately 30%. In CN-AML with poorer prognosis, mutations in the de novo DNA methyltransferase (DNMT3A) and the FMS-like tyrosine kinase 3 (Flt3) commonly co-occur (1-3). We demonstrate that mice with Flt3-internal-tandem duplication (Flt3ITD) and inducible deletion of Dnmt3a spontaneously develop a rapidly-lethal, completely-penetrant, and transplantable AML of normal karyotype. These murine AML retain a single Dnmt3a floxed allele, revealing the oncogenic potential of Dnmt3a haploinsufficiency. FLT3-ITD/DNMT3A-mutant primary human and murine AML demonstrate a similar pattern of global DNA methylation. In the murine model, rescuing DNMT3A expression was accompanied by DNA re-methylation and loss of clonogenic potential, suggesting that Dnmt3a-mutant oncogenic effects are reversible. Differentially methylated genomic regions were associated with changes in the expression of nearby genes. Moreover, dissection of the cellular architecture of the AML model using single-cell RNA-Seq, flow cytometry and colony assays identified clonogenic subpopulations that differentially express genes that are sensitive to the methylation of nearby genomic loci and varied in response to Dnmt3a levels. Thus, Dnmt3a haploinsufficiency transforms Flt3ITD myeloproliferative disease by modulating methylation-sensitive gene expression within a clonogenic AML subpopulation. To identify the gene expression changes associated with Dnmt3a loss of function in human and murine Flt3-ITD and Dnmt3a-mutant AML (Bulk RNA-Seq).
</TABLE>


<TABLE border=1>
<TR><TH>NO_Word<TH>count_in_desc
<TR><TD>cell<TD>1
<TR><TH>YES_Word<TH>count_in_desc
<TR><TD>mutant<TD>3
</TABLE>
<br><br>




False Positive: <b>E-MTAB-3814</b><p>


<TABLE border=1>
<TR><TH>accid<TH>name
<TR><TD>E-MTAB-3814<TD>Transcriptional profiling of colonic mouse fibroblasts stimulated with interleukin-36 gamma
</TABLE>
<p>
<TABLE border=1>
<TR><TH>NO_Word<TH>count_in_name
<TR><TD colspan=2>0 NO Words in name
<TR><TH>YES_Word<TH>count_in_name
<TR><TD colspan=2>0 YES Words in name
</table>
<p>

<TABLE border=1>
<TR><TH>accid<TH>description
<TR><TD>E-MTAB-3814<TD>Intestinal fibroblasts from the colons of C57BL/6 wild-type mice were grown in the presence or absence of IL-36ÉŁ (30 ng/ml) for 4 hours. The experimental setup included 6 samples from 3 colons in a pairwise design (3 stimulated vs 3 unstimulated). Profiling of whole-genome transcriptomic patterns was performed with 1.2 Âľg high quality total RNA per sample by RNA-seq at the Next Generation Sequencing core unit of the university hospital Erlangen. The sequencing platform was Illumina HiSeq-2500 with high output mode and single-end 100bp fragments. The sequencing library was NuGEN Ovation RNA-Seq Library Prep-Kit. Demultiplexed reads were corrected regarding rRNAs, tRNAs, mt-rRNAs and mt-tRNAs. Alignment against the mus musculus reference genome (Ensemble version 75 for GRCm38) was performed with RNAseq aligner STAR (version 2.4.0.i) and quantification (HTseq count) was done of unique mappings. 
</TABLE>


<TABLE border=1>
<TR><TH>NO_Word<TH>count_in_desc
<TR><TD colspan=2>0 NO Words in desc
<TR><TH>YES_Word<TH>count_in_desc
<TR><TD>design<TD>1
<TR><TD>wild<TD>1
</TABLE>
<br><br>






False Positive: <b>E-GEOD-31013</b><p>


<TABLE border=1>
<TR><TH>accid<TH>name
<TR><TD>E-GEOD-31013<TD>Transcription profiling by array of spontaneous lung tumors in mice to compare to human non-small cell lung cancer
</TABLE>
<p>
<TABLE border=1>
<TR><TH>NO_Word<TH>count_in_name
<TR><TD>cell<TD>1
<TR><TH>YES_Word<TH>count_in_name
<TR><TD colspan=2>0 YES Words in name
</table>
<p>

<TABLE border=1>
<TR><TH>accid<TH>description
<TR><TD>E-GEOD-31013<TD>Introduction: Lung cancer is the leading cause of cancer-related death in people. There are several chemically induced and genetically modified mouse models used to study lung cancer. We hypothesized that spontaneous murine (B6C3F1) lung tumors can serve as a model to study human non-small cell lung cancer (NSCLC). Methods: RNA was extracted from untreated 2-year-old B6C3F1 mouse spontaneous lung (SL) tumors and age-matched normal lung tissue from a chronic inhalation NTP study. Global gene expression analysis was performed using Affymetrix Mouse Genome 430 2.0 GeneChip® arrays. After data normalization, for each probe set, pairwise comparisons between groups were made using a bootstrap t-test while controlling the mixed directional false discovery rate (mdFDR) to generate a differential gene expression list. IPA, KEGG, and EASE software tools were used to evaluate the overrepresented cancer genes and pathways. Results: MAPK and TGF-beta pathways were overrepresented within the dataset. Almost all of the validated genes by quantitative real time RT-PCR had comparable directional fold changes with the microarray data. The candidate oncogenes included Kras, Braf, Raf1, Id2, Hmga1, Cks1b, and Foxf1. The candidate tumor suppressor genes included Rb1, Cdkn2a, Hnf4a, Tcf21, Ptprd, Hpgd, Hopx, Ogn, Id4, Hoxa5, Smad6, Smad7, Zbtb16, Cyr61, Dusp4, and Ifi16. In addition, several genes important in lung development were also differentially expressed, such as Smad6, Hopx, Sox4, Sox9 and Mycn. Conclusion: In this study, we have demonstrated that several cancer genes and signaling pathways relevant for human NSCLC were similarly altered in spontaneous murine lung tumors. Six spontaneous lung tumors and six normal lungs (as controls) from 2-year-old B6C3F1 mice.
</TABLE>


<TABLE border=1>
<TR><TH>NO_Word<TH>count_in_desc
<TR><TD>cell<TD>1
<TR><TD>induced<TD>1
<TR><TH>YES_Word<TH>count_in_desc
<TR><TD>development<TD>1
</TABLE>
<br><br>




False Positive: <b>E-GEOD-2882</b><p>


<TABLE border=1>
<TR><TH>accid<TH>name
<TR><TD>E-GEOD-2882<TD>Transcription profiling of microdissected and hand sorted mouse forebrain neurons to understand the extend and nature of differential expression in various cell types
</TABLE>
<p>
<TABLE border=1>
<TR><TH>NO_Word<TH>count_in_name
<TR><TD>cell<TD>1
<TR><TH>YES_Word<TH>count_in_name
<TR><TD colspan=2>0 YES Words in name
</table>
<p>

<TABLE border=1>
<TR><TH>accid<TH>description
<TR><TD>E-GEOD-2882<TD>The mammalian forebrain is a tissue of stunning complexity comprised of numerous regions each containing many distinct cell types that differ in their intrinsic and synaptic physiology, morphology and connectivity. These differences are likely conferred by differential gene expression, but the extent and nature of cell type specific gene expression is largely unknown. Here, we carried out microarray analysis of twelve major classes of fluorescently labelled neurons within the forebrain and provide the first comprehensive view of gene expression differences. The results demonstrate a profound molecular heterogeneity among neuronal subtypes, represented disproportionately by gene paralogs, and begin to reveal the genetic programs underlying the fundamental divisions between neuronal classes including that between glutamatergic and GABAergic neurons. Experiment Overall Design: Total of 36 samples from 12 different cell types were anlyzed. Three biological replicates (each sample from different animal(s)) were analyzed for each cell type.
</TABLE>


<TABLE border=1>
<TR><TH>NO_Word<TH>count_in_desc
<TR><TD>cell<TD>4
<TR><TH>YES_Word<TH>count_in_desc
<TR><TD>design<TD>1
</TABLE>
<br><br>




False Positive: <b>E-GEOD-9033</b><p>


<TABLE border=1>
<TR><TH>accid<TH>name
<TR><TD>E-GEOD-9033<TD>Transcription profiling of mouse and hamster radiation hybrids to map regulatory loci for protein coding genes
</TABLE>
<p>
<TABLE border=1>
<TR><TH>NO_Word<TH>count_in_name
<TR><TD colspan=2>0 NO Words in name
<TR><TH>YES_Word<TH>count_in_name
<TR><TD colspan=2>0 YES Words in name
</table>
<p>

<TABLE border=1>
<TR><TH>accid<TH>description
<TR><TD>E-GEOD-9033<TD>We have mapped regulatory loci for nearly all protein coding genes in the mammalian genome using microarray measurements from a panel of mouse/hamster radiation hybrids. Experiment Overall Design: 99 RH cells with biological replicates (198 total) and biological replicates of mouse brain RNA vs. hamster brain RNA and mouse liver RNA vs. hamster liver RNA (4 total)
</TABLE>


<TABLE border=1>
<TR><TH>NO_Word<TH>count_in_desc
<TR><TD>cells<TD>1
<TR><TH>YES_Word<TH>count_in_desc
<TR><TD>design<TD>1
</TABLE>
<br><br>





False Positive: <b>E-GEOD-47735</b><p>


<TABLE border=1>
<TR><TH>accid<TH>name
<TR><TD>E-GEOD-47735<TD>Interplay of host microbiota, genetic perturbations, and inflammation promote local development of intestinal neoplasms in mice
</TABLE>
<p>
<TABLE border=1>
<TR><TH>NO_Word<TH>count_in_name
<TR><TD colspan=2>0 NO Words in name
<TR><TH>YES_Word<TH>count_in_name
<TR><TD>development<TD>1
</table>
<p>

<TABLE border=1>
<TR><TH>accid<TH>description
<TR><TD>E-GEOD-47735<TD>The preferential localization of some neoplasms, such as serrated polyps, in specific areas of the intestine suggests that non-genetic factors may be important for their development. To test this hypothesis, we took advantage of transgenic mice that expressed HB-EGF throughout the intestine, but develop serrated polyps only in the cecum. Here we show that a host-specific microbiome was associated with serrated polyps, and that alterations of the microbiota induced by antibiotic treatment or by embryo-transfer rederivation markedly inhibited the formation of serrated polyps in the cecum. Mechanistically, development of serrated polyps was associated with a local decrease in epithelial barrier-function, bacterial invasion, production of antimicrobials, and increased expression of several inflammatory factors such as IL-17, Cxcl2, Tnf-Œ±, and IL-1. Increased number of neutrophils were found within the serrated polyps, and their depletion significantly reduced polyp growth. Together these results indicate that non-genetic factors contribute to the development of serrated polyps and suggest that the development of these intestinal neoplasms in the cecum is driven by the interplay between genetic changes in the host, an inflammatory response, and a host-specific microbiota. SUMMARY: Serrated polyps (SP) are a heterogeneous group of neoplasms found in particular areas of the gut. To define the factors contributing to their specific localization, we analyzed a strain of transgenic mice that carry a genetic alteration throughout the intestinal epithelium, but only develop SP in the cecum. Transcriptome and immunostaining analyses showed increased expression of antimicrobial genes, inflammatory factors, and the presence of bacteria within SP. Alteration of the cecal microbiota by antibiotic treatment or by embryo-transfer rederivation dramatically reduced SP incidence. Microbiome analysis implicated a limited set of bacteria in the development of SP. Together, these results point to a crucial role for the microbiota in the localized development of SP in a genetically susceptible host. We obtained serrated polyp (SP) and surrounding normal (NM) tissue from the ceca of three affected mice (paired design) and assessed expression differences by RNA-Seq.
</TABLE>


<TABLE border=1>
<TR><TH>NO_Word<TH>count_in_desc
<TR><TD>induced<TD>1
<TR><TH>YES_Word<TH>count_in_desc
<TR><TD>design<TD>1
<TR><TD>development<TD>6
</TABLE>
<br><br>






False Positive: <b>E-GEOD-47395</b><p>


<TABLE border=1>
<TR><TH>accid<TH>name
<TR><TD>E-GEOD-47395<TD>RNA Seq analysis of Mefs Isolated from STC1+/+ and STC1-/- mice
</TABLE>
<p>
<TABLE border=1>
<TR><TH>NO_Word<TH>count_in_name
<TR><TD colspan=2>0 NO Words in name
<TR><TH>YES_Word<TH>count_in_name
<TR><TD colspan=2>0 YES Words in name
</table>
<p>

<TABLE border=1>
<TR><TH>accid<TH>description
<TR><TD>E-GEOD-47395<TD>Evaluation of X-linked gene expression in MEFs isolated from STC1 wild type and knock out mice X chromosome inactivation (XCI) is initiated in cis by the Xist RNA, which coats the inactive X chromosome (Xi) from which it is produced. We performed a large-scale RNA interference screen to identify trans-acting XCI factors (XCIFs) that comprise regulators of cell signaling and transcription. We find that the XCIFs promote Xist expression and/or localization to the Xi. One of the XCIFs, STC1, is a glycoprotein found in both the cytoplasm and nucleus and whose function is poorly understood. A homozygous mouse knockout of STC1 has a defect in XCI but surprisingly is phenotypically normal. We performed transcriptome profiling (RNA-Seq) experiments to determine whether the expression levels of X-encoded genes were elevated in Stc1-/- female MEFs. In these experiments, RNA was prepared from three independent cultures of Stc1+/+ or Stc1-/- female MEFs. RNA samples were processed and amplified followed by deep sequencing. The similarity of X-linked gene expression between Stc1+/+ and Stc1-/- MEFs was statistically significant. The vast majority of autosomal genes were also expressed at comparable levels in Stc1+/+ and Stc1-/- MEFs. Sequenced mRNA isolated from the STC1+/+ or STC1-/- MEFs.
</TABLE>


<TABLE border=1>
<TR><TH>NO_Word<TH>count_in_desc
<TR><TD>cell<TD>1
<TR><TH>YES_Word<TH>count_in_desc
<TR><TD>knockout<TD>1
<TR><TD>produced<TD>1
<TR><TD>wild<TD>1
</TABLE>
<br><br>







False Positive: <b>E-GEOD-15729</b><p>


<TABLE border=1>
<TR><TH>accid<TH>name
<TR><TD>E-GEOD-15729<TD>Gene Expression of ApoE Null and ApoE Null/RAGE Diabetic and Non-diabetic Mice
</TABLE>
<p>
<TABLE border=1>
<TR><TH>NO_Word<TH>count_in_name
<TR><TD colspan=2>0 NO Words in name
<TR><TH>YES_Word<TH>count_in_name
<TR><TD colspan=2>0 YES Words in name
</table>
<p>

<TABLE border=1>
<TR><TH>accid<TH>description
<TR><TD>E-GEOD-15729<TD>The multi-ligand Receptor for AGE (RAGE) contributes to atherosclerosis in apolipoprotein (ApoE) null mice in both the non-diabetic and diabetic states. Previous studies using soluble RAGE, the extracellular ligand-binding domain of RAGE, or homozygous RAGE null mice showed that blockade or deletion of RAGE resulted in marked reduction in atherosclerotic lesion area and complexity compared to control animals. In parallel, significant down-regulation of inflammatory mediators and matrix metalloproteinases was evident in ApoE null mice aortas devoid of RAGE compared to those of ApoE null RAGE-expressing mice. Although these findings suggested that RAGE triggered pro-atherogenic mechanisms via regulation of inflammatory gene expression, these studies did not reveal the broader pathways by which RAGE contributed to atherosclerosis in ApoE null mice. Therefore, we performed Affymetrix gene expression arrays on aortas of non-diabetic and diabetic ApoE null mice expressing RAGE or devoid of RAGE at nine weeks of age, as this reflected a time point at which frank atherosclerotic lesions were not yet present, but, that we would be able to identify the genes likely involved in diabetes- and RAGE-dependent atherogenesis. The comparisons were as follows: 1. diabetic ApoE null relative to non-diabetic ApoE null; 2. non-diabetic ApoE null / RAGE null relative to non-diabetic ApoE null; 3. diabetic ApoE null / RAGE null relative to non-diabetic ApoE null / RAGE null; and 4. diabetic ApoE null / RAGE null relative to diabetic ApoE null aorta. Our data reveal that there is very little overlap of the genes which are differentially expressed both in the onset of diabetes in ApoE null mice, and in the effect of RAGE deletion in diabetic ApoE null mice. We next performed a Pathway-Express analysis to determine the pathways that were most associated with the onset of diabetes in ApoE null mice and the effect of RAGE gene deletion in diabetic ApoE null mice. Rigorous statistical analysis was undertaken and revealed that the transforming growth factor-beta pathway (tgf-beta) and focal adhesion pathways might be expected to play a significant role in both the mechanism by which diabetes facilitates the formation of atherosclerotic plaques in ApoE null mice, and the mechanism by which deletion of RAGE ameliorates this effect. We focused on three genes of the tgf-betafamily which were found to be up-regulated in diabetic vs. non-diabetic ApoE null aorta, and which were reduced by deletion of RAGE. These included: thrombospondin1 (Thbs1), transforming growth factor-beta (tgf-beta) and rho-associated kinase (ROCK1). Real-time quantitative polymerase chain reaction and Western blotting experiments were performed, as well as ROCK1 activity assays in mouse aorta, and validated the findings of the Affymetrix gene array. Further, confocal microscopy revealed that a principal cell type in the ApoE null aorta expressing these factors was the vascular smooth muscle cell. Our data suggest the novel finding that the observed reduction of accelerated atherosclerosis in diabetic ApoE null / RAGE null vs. diabetic ApoE null mice occurs, all or in part, through the ROCK1 branch of the TGF-betapathway. We have inferred a detailed mechanism for this process. Taken together, these data suggest that suppression of ROCK1 activity in the atherosclerosis-vulnerable vessel wall, especially in diabetes, but in non-diabetes as well, may underlie the beneficial effects of RAGE antagonism and genetic deletion in murine models. These findings highlight logical and novel targets for therapeutic intervention in cardiovascular disease and diabetes. 1. diabetic ApoE null relative to non-diabetic ApoE null; 2. non-diabetic ApoE null / RAGE null relative to non-diabetic ApoE null; 3. diabetic ApoE null / RAGE null relative to non-diabetic ApoE null / RAGE null; and 4. diabetic ApoE null / RAGE null relative to diabetic ApoE null aorta. There were 4 mice in each group initially. However there are only 3 non-diabetic ApoE null / RAGE null mice in the final experimental sample in group 3 due to a failure to generate cRNA from that sample. All samples were normalized to remove chip-dependent regularities using the RMA method. Chips and controls at each combination of genotype and disease sate were normalized together. The statistical significance of differential expression was calculated using the empirical Bayesian LIMMA (LInear Model for MicroArrays) method A cut-off B>0 was used for the statistical significance of gene expression.
</TABLE>


<TABLE border=1>
<TR><TH>NO_Word<TH>count_in_desc
<TR><TD>cell<TD>2
<TR><TH>YES_Word<TH>count_in_desc
<TR><TD>genotype<TD>1
</TABLE>
<br><br>







False Positive: <b>E-GEOD-66420</b><p>


<TABLE border=1>
<TR><TH>accid<TH>name
<TR><TD>E-GEOD-66420<TD>Defining the microglia transcriptome in multifunctional protein-2 deficient mice
</TABLE>
<p>
<TABLE border=1>
<TR><TH>NO_Word<TH>count_in_name
<TR><TD>cell<TD>1
<TR><TH>YES_Word<TH>count_in_name
<TR><TD colspan=2>0 YES Words in name
</table>
<p>

<TABLE border=1>
<TR><TH>accid<TH>description
<TR><TD>E-GEOD-66420<TD>Purpose: We purified whole brain microglia of MFP2 knockout mice and control mice utilizing percoll gradient and FACS sorting, followed by microarray analysis to define the molecular changes in MFP2 knockout mice at the endstage of the disease. We compared the microglia transcriptome of Mfp2-/- microglia to that of SOD1-G93A microglia isolated from spinal cord to define the microglia signature associated with a non-neurodegenerative environment. Results and conclusions: Mfp2-/- microglia acquire an activation state characterized by activation of mammalian target of rapamycin (mTOR). In addition, activated microglia display reduced expression of genes that are normally highly expressed by surveillant microglia in steady-state conditions. The immunological profile of is heterogeneous and encompasses upregulation of both pro- and anti-inflammatory genes. In contrast to the neurodegeneration-specific microglia profile in SOD1-G93A mice, Mfp2-/- microglia do not induce genes associated with phagocytosis, lysosomal activation and neurotoxicity. 4 MFP2 knockout and 4 control samples were subjected to microarray analysis.
</TABLE>


<TABLE border=1>
<TR><TH>NO_Word<TH>count_in_desc
<TR><TD>facs<TD>1
<TR><TH>YES_Word<TH>count_in_desc
<TR><TD>knockout<TD>3
</TABLE>
<br><br>






















</OL>
</OL>

</body>
</html>